src/training/cli.py

```
@@ -0,0 +1,180 @@
+        import subprocess
+
+        script_path = Path(__file__).parent.parent.parent / "scripts" / "training" / "minimal_working_training.py"
+        # Resolve to absolute path and validate it's within the project directory
+        script_path = script_path.resolve()
+        project_root = Path(__file__).parent.parent.parent.resolve()
+
+        if script_path.exists() and script_path.is_file() and script_path.is_relative_to(project_root):
+            cmd = [sys.executable, str(script_path)]
+            if args.gpu:
+                cmd.append("--gpu")
+            subprocess.run(cmd, check=True)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)